Rocket.Chat/de.lproj/Localizable.strings

```diff
@@ -83,6 +83,7 @@
 "user_menu.away" = "Abwesend";
 "user_menu.busy" = "Beschäftigt";
 "user_menu.invisible" = "Unsichtbar";
+"user_menu.admin" = "Administration"; //TODO
```

Comment:
No need for //TODO ;)

Reply:
![](https://open.rocket.chat/emoji-custom/ninja.gif)